packages/scaffolding-cli/src/domain/config/questions.ts

```diff
@@ -130,25 +130,25 @@ export function cliAdvancedQuestions(): Array<PromptQuestion> {
         {
             "type": "text",
             "name": "business_company",
-            "message": "Please provide the company name (will be used namespacing conventions)",
+            "message": "Please provide the company name (will use namespacing conventions)",
```

Comment:
It's used for renaming the namespaces for the dotnet projects. eg `xxAMIDOxx.xxSTACKSxx.E2E.Selenium` becomes `business_company.business_project.E2E.Selenium`

Reply:
Do I need to change the wording to reflect this? Not 100% sure what it means in context.

Reply:
oh yeah this one should be probably be 
`"message": "Please provide the company name (will be used for namespacing conventions)",`
If that makes sense - not even spell check is allowing  - but yeah like @estherthetesteramido  said...